From 53aff8d4da4230619ecd2c029e2b991dccd2e47b Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Mon, 12 Jul 2010 01:52:39 -0400
Subject: [PATCH] Look for VBoxKeyboard files wherever were they created

*_INST causes the files to be created in PATH_INS, not PATH_TARGET. This
fixes a build failure when the two are different.

tar --owner 0 --group 0 --ignore-failed-read -cjRf /br/VirtualBox-3.2.6_OSE/obj/bin/vboxkeyboard.tar.bz2 \
    -C /br/VirtualBox-3.2.6_OSE/out/linux.x86/release/obj/VBoxKeyboard/install VBoxKeyboard
tar: /br/VirtualBox-3.2.6_OSE/out/linux.x86/release/obj/VBoxKeyboard/install: Cannot chdir: No such file or directory
tar: Error is not recoverable: exiting now
---
 .../Frontends/Common/VBoxKeyboard/Makefile.kmk     |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/VBox/Frontends/Common/VBoxKeyboard/Makefile.kmk b/src/VBox/Frontends/Common/VBoxKeyboard/Makefile.kmk
index b88b1c1..9d57e09 100644
--- a/src/VBox/Frontends/Common/VBoxKeyboard/Makefile.kmk
+++ b/src/VBox/Frontends/Common/VBoxKeyboard/Makefile.kmk
@@ -45,12 +45,12 @@ VBoxKeyboard_LIBPATH  = $(VBOX_LIBPATH_X11)
 # but wtf, it's not like we're even trying to be FHS compliant).
 #
 ## @todo this should be tar'ed down into the output directory and installed using an install target.
-VBOX_KEYBOARD_STAGE_DIR = $(PATH_TARGET)/VBoxKeyboard/install
+VBOX_KEYBOARD_STAGE_DIR = $(PATH_INS)/VBoxKeyboard/install
 VBOX_KEYBOARD_DEST_DIR = $(VBOX_KEYBOARD_STAGE_DIR)/VBoxKeyboard/
 
 INSTALLS += VBoxKeyboard-Files
 
-VBoxKeyboard-Files_INST    = $(subst $(PATH_TARGET),obj,$(VBOX_KEYBOARD_DEST_DIR))
+VBoxKeyboard-Files_INST    = $(subst $(PATH_INS)/,,$(VBOX_KEYBOARD_DEST_DIR))
 VBoxKeyboard-Files_MODE    = a+rx,u+w
 VBoxKeyboard-Files_SOURCES = \
 		COPYING.LIB \
-- 
1.7.1

